documentation/CHANGES.rst

```diff
@@ -514,3 +544,6 @@ Known Issues and Limitations
    ``std::less`` or ``std::greater``, otherwise Merge sort.
 .. _`the oneDPL Specification`: https://spec.oneapi.com/versions/latest/elements/oneDPL/source/index.html
 .. _`Intel® oneAPI Threading Building Blocks (oneTBB) Release Notes`: https://software.intel.com/content/www/us/en/develop/articles/intel-oneapi-threading-building-blocks-release-notes.html
+.. _`the Library Guide`: https://oneapi-src.github.io/oneDPL/index.html
```

Comment:
Please change to oneDPL Guide.

Reply:
Please remove "the" in the links above/below.

Reply:
Done